modules/ROOT/pages/reports.adoc

```diff
@@ -6,11 +6,13 @@ endif::[]
 *https://www.mulesoft.com/anypoint-pricing[This feature requires a Titanium subscription.]*
 
 == About Reports
-Reports allow you to view a list of all the CloudHub apps running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
+Reports allow you to view a list of apps running in an environment. The reports display information about overall performance and behavior across all the apps in the environment. These reports can help you identify app performance and availability problems across the environment so you can better maintain the expected level of service for your apps.
 
 From any of the *Reports* tabs, you can:
 
-* Navigate to reports that display information about requests, performance, failures, and CPU and memory utilization.
+* Navigate to reports that display the following information:
+** Requests reports, Performance reports, and Failures reports are available for all deployment types.
```

Comment:
If "Requests", "Performance", and "Failures" are in the UI, I would make these bold:
```
** All deployment types provide *Requests* reports, *Performance* reports, and *Failures* reports.
```
If they're not, I would change the capitalization and plurals slightly:
```
** All deployment types provide request reports, performance reports, and failure reports.
```

Reply:
```suggestion
** All deployment types provide *Requests* reports, *Performance* reports, and *Failures* reports.
```